app/models/metric/chargeback_helper.rb

```diff
@@ -22,7 +22,15 @@ def tag_list_with_prefix
     if resource.kind_of?(Container)
       state = resource.vim_performance_state_for_ts(timestamp.to_s)
       image_tag_name = "#{state.image_tag_names}|" if state
-      labels = resource.try(:container_image).try(:docker_labels).try(:collect) { |l| "container_image/label/managed/#{l.name}/#{l.value}" }
+      labels = resource.try(:container_image).try(:docker_labels).try(:collect) do |l|
+        base64_name = "#{AssignmentMixin.const_get(:BASE64_PREFIX)}:{#{Base64.strict_encode64(l.name)}}"
+        base64_value = "#{AssignmentMixin.const_get(:BASE64_PREFIX)}:{#{Base64.strict_encode64(l.value)}}"
+        [
+          # The assignments in tags can appear the old way, as they are or in base64
+          "container_image/label/managed/#{l.name}/#{l.value}",
+          "container_image/label/managed/#{base64_name}/#{base64_value}"
+        ]
+      end.flatten
     end
```

Comment:
here we are matching an images labels with what it present in the DB. they could appear either in base64 or just as they were.